assets/constants.ts

```diff
@@ -5580,6 +5580,10 @@ export default {
   "OPPORTUNITY_TYPE_TRANSLATION": "translation",
   "OPPORTUNITY_TYPE_VOICEOVER": "voiceover",
 
+  // The bucket name used by gcs to store files when development server
+  // is running.
+  "DEFAULT_GCS_RESOURCE_BUCKET_NAME": "app_default_bucket",
```

Comment:
Don't we already have a GCS_RESOURCE_BUCKET_NAME constant in this file?

Reply:
That is prod bucket name, if we want to use GCS in the frontend in local development server then we need to use this bucket name.

Reply:
Ah ok, could you please put the two constants near each other in the file and provide enough comments to make clear what the differences are? I think that would help.

Reply:
Its right above the other constant. I improved the comment, though.

Reply:
Ah, I see the issue. In that case, could you please just use the existing constant but change its value to "app_default_bucket"? We already update that value when deploying to prod, so no point having two different constants for the same thing -- it'll just cause confusion.

Reply:
Done.